osquery/utils/conversions/windows/tests/strings.cpp

```diff
@@ -14,7 +14,21 @@
 
 namespace osquery {
 
-class ConversionsTests : public testing::Test {};
+class ConversionsTests : public testing::Test {
+ public:
+  ConversionsTests() {}
+
+  void SetUp() {
+    auto ret = CoInitializeEx(0, COINIT_MULTITHREADED);
```

Comment:
What happens if this was already called, for example in the `main()` function before tests are discovered and dispatched?

Reply:
It's ok for `CoInitializeEx` to be called multiple times, as long as it's by the same thread. MSDN doesn't specify what happens if you call it multiple times from different threads, but I don't think that's a concern here for tests